tee-worker/core/rpc-client/src/mock.rs

```diff
@@ -95,6 +95,10 @@ impl DirectApi for DirectClientMock {
 		RuntimeMetadataPrefixed::decode(&mut metadata.as_slice()).map_err(|e| e.into())
 	}
 
+	fn get_state_metadata_raw(&self) -> Result<String> {
+		unimplemented!()
```

Comment:
Have you considered returning `self.metadata` or maybe we don't need this mock ?